src/vconsole/vconsole-setup.c

```diff
@@ -330,3 +330,5 @@ int main(int argc, char **argv) {
 
         return font_ok && keyboard_ok ? EXIT_SUCCESS : EXIT_FAILURE;
 }
+
+/* vim: set ts=8 sts=8 sw=8 et : */
```

Comment:
please do not add this. We have a global set of vim settings in the project root dir in .vimrc. There's also .dir-locals.el for Emacs, and .editorconfig for everybody else. There's really no need to add modelines to the source files themselves. Just enable interpretation of vimrc files in your editor...